charts/kashti/templates/deployment.yaml

```diff
@@ -25,6 +25,10 @@ spec:
         app: {{ template "name" . }}
         release: {{ .Release.Name }}
     spec:
+      {{- if .Values.kashti.affinity }}
```

Comment:
I'd favor config being at the root level, so `.Values.affinity` instead of under `.Values.kashti.*`